extensions/wikia/AbTesting/js/AbTest.js

```diff
@@ -207,6 +207,9 @@
 	// Returns true if value falls into provided ranges
 	function isInRanges( value, ranges ) {
 		var i, range;
+		if ( !ranges || !ranges.length ) {
```

Comment:
you don't have to check the `ranges.length`
If length is 0 then it won't go into the `for` loop and it will return `false`.

You can consider using `Array.isArray` method - https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray